src/Symfony/Component/PropertyAccess/PropertyPath.php

```diff
@@ -51,6 +51,14 @@ class PropertyPath implements \IteratorAggregate, PropertyPathInterface
      */
     private $isIndex = [];
 
+    /**
+     * Contains a Boolean for each property in $elements denoting whether this
+     * element is optional or not.
+     *
+     * @var array
+     */
+    private $isNullSafe = [];
```

Comment:
Should this be a native array type?

Reply:
@b1rdex I did not change that to avoid having the native type only for this property

If I add it here, I think I must add it for every parameters. Can I do it?